Cargo.toml

```diff
@@ -66,5 +66,5 @@ exclude = [
 ]
 
 [patch.crates-io]
-fluent-templates = { git = "https://github.com/robjtede/fluent-templates.git", branch = "fix-circ-dep" }
-fluent-template-macros = { git = "https://github.com/robjtede/fluent-templates.git", branch = "fix-circ-dep" }
+fluent-templates = { git = "https://github.com/XAMPPRocky/fluent-templates.git"}
+fluent-template-macros = { git = "https://github.com/XAMPPRocky/fluent-templates.git"}
```

Comment:
Let's pin the commit and add FIXME not to forget why we cannot just use them on crates.io.

```suggestion
# FIXME: Specify commit until https://github.com/XAMPPRocky/fluent-templates/pull/36 is released
fluent-templates = { git = "https://github.com/XAMPPRocky/fluent-templates.git", rev = "82b50baf6232e1ce2e9629ce2a8fcc12ab5d942c" }
fluent-template-macros = { git = "https://github.com/XAMPPRocky/fluent-templates.git", rev = "82b50baf6232e1ce2e9629ce2a8fcc12ab5d942c" }
```